### DISCUSSION TITLE
folders batch renaming

### DISCUSSION THREAD

**Discussion by d0ublezer0** [mikekazakov/nimble-commander#349] [Q&A] [answered]
<img width="683" alt="image" src="https://github.com/user-attachments/assets/51cf1083-94ff-40eb-8ba1-fb2c85a704ca">

batch renaming dialog adds dots to folders.
is is correct?

**Comment by mikekazakov** [comment, marked as answer]
Yes, that is the correct behaviour - the Batch Rename follows the requested mask, which contains the dot symbol in this screenshot.
